src/NotificationTargetCommonITILObject.php

```diff
@@ -147,6 +147,7 @@ public function getEvents()
             'pendingreason_add' => __('Pending reason added'),
             'pendingreason_del' => __('Pending reason removed'),
             'pendingreason_close' => __('Pending reason auto close'),
+            'del_assign_user'   => __('Deletion of a user in assignees'),
```

Comment:
IMHO, we should add a `remove_xxx` event for all possible actor type (e.g. requester user, requester group, observer user, observer group, assigned user, assigned group and assigned suplier).

If we add this only for assigned users, I am pretty sure that someone will ask for the same feature for another actor type as soon as the feature will be released.